types/src/proof/position/mod.rs

```diff
@@ -446,13 +446,13 @@ impl Iterator for FrozenSubtreeSiblingIterator {
     }
 }
 
-pub fn children_of_node(node: u64) -> u64 {
```

Comment:
These functions expose the internal traversal order and no longer need to be public.